src/js/play.js

```diff
@@ -114,29 +111,33 @@ var playState = {
     crackEggs: function(egg){
         switch (egg.key){
             case gameController.REGULAR_EGG:
-                gameController.regularEggChain = 0;
+                gameController.resetRegularEggStreak();
                 gameController.tweenEgg(gameController.CRACKED_REGULAR_EGG, egg);
-                gameController.eggCrack.play();
+                gameController.playEggCrackingSound();
                 break;
             case gameController.BOMB:
                 gameController.tweenEgg(gameController.BOMB_EXPLOSION_CLOUD, egg);
-                gameController.bombWhoosh.play();
+                gameController.playEggCrackingSound();
                 break;
             case gameController.FRENZY_EGG:
+                gameController.resetRegularEggStreak();
```

Comment:
Well, that's exactly why I wanted to introduce this. the idea of their streak breaking would encourage them to collect special eggs. The streak is preserved when they return back to the regular play state. But we should probably user test this and talk to Paul about it. I'm also conflicted in regards to this matter - It is good that user wants to keep on going with the regular eggs. However, as a consequence, they will never want to catch the special eggs.

Reply:
Yeah, we should probably ask paul about it & user test. Because I feel like if users don't want to collect special eggs then forcing them to collect them will probably just make them frustrated.